app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,95 @@
+<div class="signup-container">
+  <h2>Sign up</h2>
+
+  <form action="<%= registration_path(resource_name) %>" method="post">
+    <%= csrf_meta_tags %>
+
+    <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="form-group">
+      <label for="user_name">Name</label>
+      <input type="text" id="user_name" name="user[name]" placeholder="Enter your name" required>
+    </div>
+
+    <div class="form-group">
+      <label for="user_email">Email</label>
+      <input type="email" id="user_email" name="user[email]" placeholder="Enter your email" required>
+    </div>
+
+    <div class="form-group">
+      <label for="user_password">Password</label>
+      <input type="password" id="user_password" name="user[password]" placeholder="Enter your password" required>
+    </div>
+
+    <div class="form-group">
+      <label for="user_password_confirmation">Confirm Password</label>
+      <input type="password" id="user_password_confirmation" name="user[password_confirmation]" placeholder="Confirm your password" required>
+    </div>
+
+    <div class="form-group">
+      <input type="submit" value="Sign up">
+    </div>
+  </form>
+
+  <%= render "devise/shared/links" %>
+</div>
+
+<style>
+  /* Main container */
+  .signup-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    background-color: #f1f1f1;
+  }
+
+  /* Form fields */
+  .form-group {
+    margin-bottom: 20px;
+    width: 100%;
+    max-width: 400px;
+  }
+
+  .form-group label {
+    display: block;
+    margin-bottom: 5px;
+  }
+
+  .form-group input[type="text"],
+  .form-group input[type="email"],
+  .form-group input[type="password"] {
+    width: 94%;
+    padding: 10px;
+    border-radius: 5px;
+    border: 1px solid #ced4da;
+  }
+
+  /* Submit button */
+  .signup-container input[type="submit"] {
+    width: 100%;
+    padding: 10px;
+    border-radius: 5px;
+    background-color: #007bff;
+    color: #fff;
+    font-size: 16px;
+    cursor: pointer;
+    transition: all 0.3s ease;
+  }
+
+  .signup-container input[type="submit"]:hover {
+    background-color: #0069d9;
+  }
+
+  /* Links */
+  .signup-container a {
+    color: #007bff;
+    text-decoration: none;
+    transition: all 0.3s ease;
+  }
+
+  .signup-container a:hover {
+    color: #0069d9;
+  }
+</style>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Good job adding all the changes recommended by the previous reviewer. However I think it would be better if you move all your styling code in a specific styling file inside your `app/assets/stylesheets/` directory. This for best practices reasons.